<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>info_grupo</class>
 <widget class="QDialog" name="info_grupo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Informaci칩n Grupo</string>
  </property>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>177</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Informaci칩n de Grupo</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btSalir">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>530</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btdiagramabarras">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>470</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Diagrama Barras</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btenviados">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>500</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Enviados</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btrecibidos">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>500</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Recibidos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bteliminarintegrante">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>470</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Eliminar Integrante</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btAbandonargrupo">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>440</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Abandonar Grupo</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Integrantes:</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listIntegrantes">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>292</width>
     <height>114</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btAgregar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>440</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>A침adir Integrante</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>68</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Descripci칩n:</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtDescripcion">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>291</width>
     <height>171</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtGrupo">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Nombre:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btSalir</sender>
   <signal>clicked()</signal>
   <receiver>info_grupo</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>272</x>
     <y>357</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>544</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
